docs/tutorials/coin_spend_rpc.md

```diff
@@ -0,0 +1,70 @@
+
+When you search for receive address, you'll see all coins secured by this puzzle. Select the one you want to spend. That will get you to coin's detail to get the amount, puzzle hash, and parent info.
+
+## Get serialized puzzle and solution
+The next thing you need to know to spend the coin is the coin's puzzle and solution. Puzzle and solution are provided in a serialized format, so we need to get that. The puzzle has to be compiled to low-level Chialisp and is serialized in "as is" format.
+
+To serialize the solution using same tools you need to slightly modify solution format to make it valid Chialisp program. For that you need to quote your solution. For example, in case of solution `(hello 0x5f5767744f91c1c326d927a63d9b34fa7035c10e3eb838c44e3afe127c1b7675 2)` you need to change it to `(q . (hello 0x5f5767744f91c1c326d927a63d9b34fa7035c10e3eb838c44e3afe127c1b7675 2))` which makes it valid Chialisp program that can be compiled. 
+
+**Example for the password-locked coin:**
+
+**Puzzle:** `(a (q 2 (i (= (sha256 5) (q . 0x2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824)) (q 4 (c 2 (c 11 (c 23 ()))) ()) (q 8)) 1) (c (q . 51) 1))`
+
+**Serialized puzzle:** `0xff02ffff01ff02ffff03ffff09ffff0bff0580ffff01a02cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b982480ffff01ff04ffff04ff02ffff04ff0bffff04ff17ff80808080ff8080ffff01ff088080ff0180ffff04ffff0133ff018080`
+
+**Solution:** `(hello 0x5f5767744f91c1c326d927a63d9b34fa7035c10e3eb838c44e3afe127c1b7675 2)` (as valid Chialisp program `(q . (hello 0x5f5767744f91c1c326d927a63d9b34fa7035c10e3eb838c44e3afe127c1b7675 2))`)
+
+**Serialized solution:** `0xff8568656c6c6fffa05f5767744f91c1c326d927a63d9b34fa7035c10e3eb838c44e3afe127c1b7675ff0280`
+
+_WARNING: You have to change this solution and replace the target puzzle hash with your own to get your coins back_
+
+### Serialization using [Quexington's Chialisp Dev Utility](https://github.com/Quexington/chialisp_dev_utility)
+
+Follow repository's README to set up a new project and serialize puzzle.
+
+In short (may change): paste your compiled puzzle/solution to your work file and call `chialisp build`. That will generate `.hex` files with a serialized version of your puzzle/solution (depending on your work file). 
+
+### Serialization using [Chialisp web tool](https://clisp.surrealdev.com/)
+
+Paste your puzzle in the text area and click **Compile**. The serialized result will be displayed in the **Serialized** section.
+
+## Spend a coin with RPC call
+To spend your coin, you only need to call RPC (broadcast transaction example) with values specific to your spend.
+
+```chialisp
```

Comment:
Probably don't use chialisp here.  I don't think it will make a difference but it is technically inaccurate.